### PR TITLE
[FileFormats.CBF] write VectorOfVariables constraints in VAR block

### PR DESCRIPTION
**Pull request by odow**
Part of #2457

### CODE REVIEW

File: src/FileFormats/CBF/write.jl [hunk 1]
```diff
@@ -125,6 +130,56 @@ function _add_cones(
...
+function _add_cones(
+    data::_CBFDataStructure,
+    model::Model,
+    ::Type{F},
+    ::Type{S},
+) where {
+    F<:MOI.VectorOfVariables,
+    S<:Union{MOI.ExponentialCone,MOI.DualExponentialCone},
+}
+    # The Exponential cone in MOI and CBF are reversed. Instead of dealing with
```
Comment (blegat) [inline src/FileFormats/CBF/write.jl:173]: I'd prefer if we say it's not supported for now. Is it the same permutation as https://github.com/jump-dev/ECOS.jl/blob/master/src/MOI_wrapper/permuted_exponential_cone.jl

**Comment by blegat** [comment]
If the PSD cone is given, do we error ?

File: src/FileFormats/CBF/write.jl (hunk 1 above)
Comment (blegat) [reply src/FileFormats/CBF/write.jl:173]: It should be the same as the Mosek exponential cone I guess. If you want it to be supported, you can add the set and bridge in this module. Then we will be able to simplify MosekTools by using this set and adding the bridge in the `ListOfNonstandardBridges`

**Comment by odow** [comment]
No change to PSD. There's still written as affine cones

File: src/FileFormats/CBF/write.jl (hunk 1 above)
Comment (odow) [reply src/FileFormats/CBF/write.jl:173]: It's not quite the same permutation as ECOS. ECOS is `1, 3, 2`, where this is `3, 2, 1`.

File: src/FileFormats/CBF/write.jl [hunk 2]
```diff
@@ -125,6 +130,56 @@ function _add_cones(
...
+    # this complexity, just write them out as `Ax + b in K` constraints.
```
Comment (odow) [inline src/FileFormats/CBF/write.jl:174]:
```suggestion
    # this complexity, just write them out as `Ax + b in K` constraints.
    # TODO(odow): we should support this at some point. See #2478
```

**Comment by blegat** [comment]
> No change to PSD. There's still written as affine cones

Ah that's the current behavior. So it goes through the scalarize bridge ?

File: src/FileFormats/CBF/write.jl (hunk 1 above)
Comment (blegat) [reply src/FileFormats/CBF/write.jl:173]: Yes, that's what I remembered, Mosek and ECOS have a different one ^^

**Comment by odow** [comment]
Before this PR, the model is written out as:
```
min c' x
Ax - b in K
x free
```
All `VectorOfVariables` constraints are accepted (no bridging) but are written as affine constraints.

After this PR, things are written as:
```
min c' x
Ax - b in K
x in some cones but not all
```
VectorOfVariables in Exponential, DualExponential, and PositiveSemidefiniteConeTriangle are still supported, but they are written as affine constraints.

**Comment by odow** [comment]
Merging as an improvement for now. I'll look at doing EXP and PSD variables in a separate PR.

**Pull request merged**